src/main/java/ch/fhnw/algd1/converters/utf8/UTF8Converter.java

```diff
@@ -3,22 +3,55 @@
 /*
  * Created on 05.09.2014
  */
+
+import java.util.Arrays;
+
 /**
  * @author 
  */
 public class UTF8Converter {
+	public static void main(String[] args) {
+		byte[] utf8Bytes = {(byte) 0xF0, (byte) 0x9F, (byte) 0x98, (byte) 0xA5};
+		int codePoint = UTFtoCodePoint(utf8Bytes);
+		System.out.println(Integer.toHexString(codePoint));
+	}
```

Comment:
(Solche Testprogramme sollte man besser in (in der Praxis zumindest) separate Klassen programmieren, damit die dann nicht in der ausgelieferten Code Bibliothek landen...)